static/docs/commands-reference/metrics.md

```diff
@@ -48,14 +48,14 @@ up and manage DVC metrics.
 First, let's create a simple DVC-file:
 
 ```dvc
-$ dvc run -d code/evaluate.py -M data/eval.json -f Dvcfile \
+$ dvc run -d code/evaluate.py -M data/eval.json \
```

Comment:
why removing this? just to simplify?

Reply:
I was reviewing the usage of `Dvcfile` everywhere (see c35037f) and I thought it was unnecessary in this doc since it's never mentioned again (and it's the default DVC-file name anyway when there's no outputs).

p.s. Thanks for prompt review! Still working on this PR as well as on issue 425, will lyk when ready for final review ASAP )

Reply:
So is this change ok, can I resolve it? Thanks